### PR TITLE
bbb-cmd: improve usb_flashdrive / add backup

### PR DESCRIPTION
**Pull request by Stadicus**
**improve usb_flashdrive**
* rename 'usb_thumbdrive' to 'usb_flashdrive'
* recognize usb drives without partitions
* improve error messages

**imlement backup of Redis dbdump and hsm_secret**
* create / restore backup of Redis database file to /mnt/backup
* create / restore backup of '/mnt/ssd/bitcoin/.lightning/hsm_secret' into Redis key lightningd:hsm_secret

### CODE REVIEW

File: armbian/base/scripts/bbb-cmd.sh [hunk 1]
```diff
@@ -16,9 +16,9 @@ possible commands:
   base restart      restarts the node
   base shutdown     shuts down the node
   
-  usb_thumbdrive    <check|mount|umount>
-  usb_backup
-
+  usb_flashdrive    <check|mount|umount>
+  backup            <create|restore|hsm_secret_create|hsm_secret_restore>
```
Comment (0xB10C) [inline armbian/base/scripts/bbb-cmd.sh]: nit: The naming here could be confusing: `bbb-cmd.sh backup hsm_secret_create`. 

> Does it create a new `hsm_secret` or does it create a new backup of the `hsm_secret` ??

Reading through the code makes it clear and I know the latter is meant, but will that be obvious to somebody else starting to work with it?

Maybe split it up into a `backup` and `restore` module? like e.g.
```
bbb-cmd.sh backup hsm_secret
bbb-cmd.sh restore hsm_secret
```

Your call.

Comment (Stadicus) [reply armbian/base/scripts/bbb-cmd.sh]: How about this?
```
bbb-cmd.sh backup sysconfig
bbb-cmd.sh backup hsm_secret
bbb-cmd.sh restore sysconfig
bbb-cmd.sh restore hsm_secret
```

Comment (0xB10C) [reply armbian/base/scripts/bbb-cmd.sh]: Sounds good!

Comment (Stadicus) [reply armbian/base/scripts/bbb-cmd.sh]: Done.

**Pull request merged**